setup.py

```diff
@@ -19,10 +19,12 @@
                       'lmdb',
                       'aiobotocore',
                       'botocore',
+                      'google-cloud-storage'
```

Comment:
how big is this dependency, should we consider making that an optional dependency?